packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -222,3 +222,23 @@ def assert_packages_match_active_buildinfo(package_ids):
 
         assert set(active_package_ids) <= set(package_ids)
         assert_packages_match_active_buildinfo(active_package_ids)
+
+
+def test_packaging_api(dcos_api_session):
+    """Test the Cosmos API (/package) wrapper
+    """
+    install_response = dcos_api_session.cosmos.install_package('kafka', '1.1.9-0.10.0.0')
```

Comment:
What we will do at that time is spin up just one cluster with the 5 agents, run tweeter (and nothing else) against that, and then tear it down. All the other tests will ignore tweeter and still use the more minimal cluster.